bin/hab-verify

```diff
@@ -66,9 +63,7 @@ function in_studio() {
   hab-origin download-sig-key "$origin"
 
   # New version of Habitat doesn't source the .studiorc
-  cmd="source .studiorc;$1"
-
-  eval "hab studio $STUDIO_OPTS run \"$cmd\""
+  eval "hab studio ${STUDIO_OPTS:-} run \"source .studiorc; $*\""
```

Comment:
I don't think there's any need for an `eval` here. This should do:
```bash
  # shellcheck disable=2086
  hab studio ${STUDIO_OPTS:-} run "source .studiorc; $*"
```
We don't want to quote `${STUDIO_OPTS:-}`, or else we'd end up with
```bash
hab studio "" run
```
which will confuse `hab studio`